### ISSUE TITLE
[drone] organization secrets require `org.` prefix

### ISSUE THREAD

**Issue by jimsheldon** [drone/go-convert#94]
Drone supports secrets at the repository and organization levels, but these secrets are not differentiated in the pipeline yaml.

Harness CI differentiates organization secrets with the `org.` prefix, see https://developer.harness.io/docs/platform/secrets/add-use-text-secrets/#step-3-reference-the-encrypted-text-by-identifier

`go-convert drone` should support taking a list of variable names and add the appropriate `org.` prefix.

For example, current converted secrets look like this:

```
<+secrets.getValue("MY_REPO_SECRET")>​
<+secrets.getValue("MY_ORG_SECRET")>​
```

We should support something like `go-convert drone --org-secrets MY_ORG_SECRET` which would convert to this:

```
<+secrets.getValue("MY_REPO_SECRET")>​
<+secrets.getValue("org.MY_ORG_SECRET")>​
```

Note: this is the identifier of the secret, not the name of the secret. As far as I can tell, with Drone yaml conversion, these are the same, but just FYI.

**Comment by bradrydzewski** [comment]
makes sense, I recommend adding `WithOrgSecrets([]string)` as optional input to handle this scenario

**Comment by jimsheldon** [comment]
Fixed in #95

Assigning this to myself to add tests.
